src/main/java/org/jenkinsci/plugins/jvctb/ViolationsToBitbucketServerRecorder.java

```diff
@@ -20,7 +20,7 @@
 import static org.jenkinsci.plugins.jvctb.perform.JvctbPerformer.jvctsPerform;
 
 public class ViolationsToBitbucketServerRecorder extends Recorder implements SimpleBuildStep {
-  @Extension
```

Comment:
forgot to remove the extension in #43 from here when it was added to the actual descriptor class.